vmware_aria_operations_integration_sdk/mp_build.py

```diff
@@ -136,20 +136,28 @@ def registry_prompt(default: str) -> str:
     )
 
 
-def get_docker_registry(adapter_kind_key: str, config_file: str) -> str:
-    docker_registry: Optional[str] = get_config_value(
-        "docker_registry", config_file=config_file
-    )
+def get_docker_registry(
+    adapter_kind_key: str,
+    config_file: str,
+    docker_registry_arg: Optional[str],
+    **kwargs,
+) -> str:
+    docker_registry = get_config_value("docker_registry", config_file=config_file)
+    default_registry_value = f"harbor-repo.vmware.com/vmware_aria_operations_integration_sdk_mps/{adapter_kind_key.lower()}"
 
     original_value = docker_registry
-    if docker_registry is None:
+    if docker_registry is None and docker_registry_arg is None:
         print(
             "mp-build needs to configure a Docker registry to store the adapter container image.",
             "class:information",
         )
-        docker_registry = registry_prompt(
-            default=f"harbor-repo.vmware.com/vmware_aria_operations_integration_sdk_mps/{adapter_kind_key.lower()}"
-        )
+        docker_registry = registry_prompt(default=default_registry_value)
+    elif docker_registry_arg is not None and not len(docker_registry_arg):
```

Comment:
If this is provided but empty, _and_ there is an entry in the config file, should we still use the default?

I wonder if it might be better to have a special argument string like "default" and not support passing `--registry-tag` with no arguments